### ISSUE TITLE
Sealed Secret Keep unsealing every 10 minutes even when --update-status enabled 

### ISSUE THREAD

**Issue by mhamaydeh** [bitnami-labs/sealed-secrets#989]
**Which component**:
The name (and version) of the affected component (controller or kubeseal)
 v0.18.5
**Describe the bug**
Sealed secret keep unsealing every approximately 10 minutes , even without applying any changes the sealed secret yaml.
even if the --update-status feature is enables , we are having the same result , it will skip update the status , but it will keep unsealing 

**To Reproduce**
Deploy sealed secret version v0.18.5 with the latest CRD. 
Create new Sealed Secret . 
Watch the Sealed Secret controller logs . or describe the sealed secret to view the events  
2. Run the command '....'
3. Wait for '....'

**Expected behavior**
Sealed secret should bot be unsealed again unless there is new changes to the sealed secret yaml file 

**Version of Kubernetes**:

- Output of `kubectl version`:

```
Server Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.5+70fb84c", GitCommit:"3c28e7a79b58e78b4c1dc1ab7e5f6c6c2d3aedd3", GitTreeState:"clean", BuildDate:"2022-04-25T15:58:12Z", GoVersion:"go1.17.5", Compiler:"gc", Platform:"linux/amd64"}


```

**Additional context**

**Comment by mhamaydeh** [comment]
Hi @alvneiayu 
here is the deployment 
`# Source: sealed-secrets/templates/deployment.yaml
apiVersion: extensions/v1beta1
kind: Deployment
metadata:
  name: sealed-secrets
  namespace: kube-system
  labels:
    app.kubernetes.io/name: sealed-secrets
    helm.sh/chart: sealed-secrets-2.2.0
    app.kubernetes.io/instance: sealed-secrets
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/version: v0.18.0
spec:
  selector:
    matchLabels:
      app.kubernetes.io/name: sealed-secrets
      app.kubernetes.io/instance: sealed-secrets
  template:
    metadata:
      labels:
        app.kubernetes.io/name: sealed-secrets
        app.kubernetes.io/instance: sealed-secrets
    spec:
      serviceAccountName: sealed-secrets-controller
      containers:
        - name: controller
          command:
            - controller
          args:
            - --key-renew-period
            - "87660h"
            - --key-prefix
            - "sealed-secrets-key"
          image: sealed-secrets-controller:v0.18.5
          imagePullPolicy: IfNotPresent
          ports:
            - containerPort: 8080
              name: http
          livenessProbe:
            failureThreshold: 3
            initialDelaySeconds: 0
            periodSeconds: 10
            successThreshold: 1
            timeoutSeconds: 1
            httpGet:
              path: /healthz
              port: http
          readinessProbe:
            failureThreshold: 3
            initialDelaySeconds: 0
            periodSeconds: 10
            successThreshold: 1
            timeoutSeconds: 1
            httpGet:
              path: /healthz
              port: http
          resources:
            limits: {}
            requests: {}
          volumeMounts:
            - mountPath: /tmp
              name: tmp
      volumes:
        - name: tmp
          emptyDir: {}
`

**Comment by mhamaydeh** [comment]
and here is the events on the secret and logs from the controller 
`
Events:
  Type    Reason    Age                  From            Message
  ----    ------    ----                 ----            -------
  Normal  Unsealed  2m35s (x5 over 35m)  sealed-secrets  SealedSecret unsealed successfully
` 

controller logs :

`
2022/10/06 11:19:18 Updating mhamaydeh/mhamaydeh-secret
2022/10/06 11:19:18 Event(v1.ObjectReference{Kind:"SealedSecret", Namespace:"mhamaydeh", Name:"mhamaydeh-secret", UID:"46e2d6c4-4565-11ed-9033-00505697f1f9", APIVersion:"bitnami.com/v1alpha1", ResourceVersion:"612010846", FieldPath:""}): type: 'Normal' reason: 'Unsealed' SealedSecret unsealed successfully
2022/10/06 11:27:30 Updating mhamaydeh/mhamaydeh-secret
2022/10/06 11:27:30 Event(v1.ObjectReference{Kind:"SealedSecret", Namespace:"mhamaydeh", Name:"mhamaydeh-secret", UID:"46e2d6c4-4565-11ed-9033-00505697f1f9", APIVersion:"bitnami.com/v1alpha1", ResourceVersion:"612010846", FieldPath:""}): type: 'Normal' reason: 'Unsealed' SealedSecret unsealed successfully

`

**Comment by alvneiayu** [comment]
hi @mhamaydeh 

We can not reproduce this problem in our Sealed Secrets deployed. We have included this PR:

https://github.com/bitnami-labs/sealed-secrets/pull/1174

And this is included in the v0.20.5. Could you test with the latest Sealed Secrets version, please.

Thanks a lot

Álvaro
